setup.py

```diff
@@ -34,15 +34,19 @@
 import sys
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import vsc_setup, log, sdw
-import vsc.mympirun.mpi.mpi as mpim
-from vsc.utils.missing import get_subclasses
+
+
+# hardcoded list, to avoid ugly hacks in order to be able to import from vsc.mympirun in setup.py...
+# this list is checked to be synced via a dedicated unit test
+MYMPIRUN_ALIASES = ['ihmpirun', 'impirun', 'm2hmpirun', 'm2mpirun', 'mhmpirun', 'mmpirun', 'ompirun']
 
 PACKAGE = {
     'install_requires': [
         'vsc-base >= 2.5.0',
         'vsc-install >= 0.10.9', # for modified subclassing
         'IPy',
     ],
+    'name': 'vsc-mympirun',
```

Comment:
OK, I looked into this more, and it seems like not specifying the package name like I'm doing here will only work if there's a `PKG-INFO` or `.git/config` file is present from which the GitHub URL can be obtained, and thus the package name.

When installing from the tarball pulled from GitHub, neither of these is there...

Doesn't it make sense to just specify the package name here, just like we do with the version? I don't see the problem with that?